screens/overviewMap.js

```diff
@@ -289,12 +383,26 @@ class OverviewMap extends PureComponent {
         >
           <Icon name="plus" family="feather" size="extraLarge" colour={colours.core.white} styles={styles.buttonIcon} />
         </TouchableOpacity>
+
+        {draggingPin && (
```

Comment:
This is interesting. What if in the popup there was an edit/delete button instead? Dragging to a point on the screen seems a bit more complicated than it needs.

Reply:
As discussed at the meeting we can't do that due to limitations with the callouts.

Reply:
That is really too bad. I wonder if there is another way to get buttons where pin is without doing it here.